### DISCUSSION TITLE
Welcome to V-Matheus Discussions!

### DISCUSSION THREAD

**Discussion by V-Matheus** [V-Matheus/V-Matheus#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by V-Matheus** [comment]
teste

**Reply by pedrohcdsouza** [reply]
reteste

**Reply by V-Matheus** [reply]
reteste do teste

**Reply by V-Matheus** [reply]
This solves the problem.
This is the solution.
The issue has been resolved.
This is the correct answer.

**Comment by pedrohcdsouza** [comment]
outro comentário

**Reply by V-Matheus** [reply]
This solves the problem.
This is the solution.
The issue has been resolved.
This is the correct answer.

**Comment by pedrohcdsouza** [comment]
mais um comentário

**Reply by V-Matheus** [reply]
This solves the problem.
This is the solution.
The issue has been resolved.
This is the correct answer.

**Comment by pedrohcdsouza** [comment]
mais outro só de ruim

**Reply by V-Matheus** [reply]
This solves the problem.
This is the solution.
The issue has been resolved.
This is the correct answer.
